src/Microsoft.DocAsCode.Build.Engine/Incrementals/IncrementalBuildContext.cs

```diff
@@ -577,6 +595,19 @@ private static string ComputeConfigHash(DocumentBuildParameters parameter, strin
             }
         }
 
+        private static string ComputeFileMetadataHash(FileMetadata fileMetadata)
+        {
+            using (new LoggerPhaseScope("ComputeFileMetadataHash", LogLevel.Diagnostic))
+            {
+                if (fileMetadata == null)
+                {
+                    return string.Empty;
+                }
+                var json = JsonUtility.Serialize(fileMetadata);
```

Comment:
>fileMetadata [](start = 49, length = 12)

do we need to consider order?

Reply:
I want to make FileMetadataHash a quick filter to determine if there is nothing changed. If only order is changed in a build, `changedGlobs` of FileMetadata will be empty, and all files can still build incrementally. So sort may not be necessary here.